src/Detector/Python/PythonDetector.cs

```diff
@@ -40,15 +40,36 @@ public PlatformDetectorResult Detect(DetectorContext context)
             var appDirectory = string.Empty;
             var hasRequirementsTxtFile = false;
             var hasPyprojectTomlFile = false;
+
             if (sourceRepo.FileExists(PythonConstants.RequirementsFileName))
             {
                 _logger.LogInformation($"Found {PythonConstants.RequirementsFileName} at the root of the repo.");
                 hasRequirementsTxtFile = true;
+
+                // Warning if missing django module
+                bool hasDjangoModule = false;
+                string filePath = $"{sourceRepo.RootPath}/{PythonConstants.RequirementsFileName}";
+                using (var reader = new StreamReader(filePath))
+                {
+                    while (!reader.EndOfStream && !hasDjangoModule)
+                    { 
+                        string line = reader.ReadLine().ToLower();
+                        if (line.StartsWith("django"))
+                        {
+                            hasDjangoModule = true;
+                        }
+                    }
+                }
+                if (!hasDjangoModule)
+                {
+                    _logger.LogWarning($"Missing django module in {PythonConstants.RequirementsFileName}");
```

Comment:
this is logging in build log .. i wonder if we need to redirect it to standard output/error as well?